mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala

```diff
@@ -216,43 +221,65 @@ class LogisticRegressionSuite extends SparkFunSuite with MLlibTestSparkContext {
   test("MultiClassSummarizer") {
     val summarizer1 = (new MultiClassSummarizer)
       .add(0.0).add(3.0).add(4.0).add(3.0).add(6.0)
-    assert(summarizer1.histogram.zip(Array[Long](1, 0, 0, 2, 1, 0, 1)).forall(x => x._1 === x._2))
+    assert(summarizer1.histogram.zip(Array[Double](1, 0, 0, 2, 1, 0, 1)).forall(x => x._1 === x._2))
```

Comment:
Is it the same as `assert(summarizer1.histogram === Array[Double](1, 0, 0, 2, 1, 0, 1)`?